mdal/mdal_logger.cpp

```diff
@@ -31,6 +35,17 @@ void MDAL::Log::warning( MDAL_Status status, std::string mssg )
   _log( MDAL_LogLevel::Warn, status, mssg );
 }
 
+void MDAL::Log::info( MDAL_Status status, std::string mssg )
```

Comment:
not sure if info will have status, just keep msg there.

and Set status always to None